app/models/concept_exercise.rb

```diff
@@ -1,5 +1,11 @@
 class ConceptExercise < Exercise
-  # TODO: Add these
-  #  has_many :exercise_concepts, class_name: "Exercise::Prerequisite", dependent: :destroy
-  #  has_many :concepts, through: :exercise_concepts, source: :track_concept
+  has_many :exercise_taught_concepts,
+    class_name: "Exercise::TaughtConcept",
+    foreign_key: :exercise_id,
+    inverse_of: :concept,
+    dependent: :destroy
```

Comment:
These might end up working fine just as straight-up join tables, but I prefer to materialise them now and then refactor them out later, rather than the other way around.